### PR TITLE
Blog post: Getting started with a core initiative

### PR DESCRIPTION
**Pull request by DavidHernandez**

### CODE REVIEW

**Comment by DavidHernandez** [comment]
Hi @RachelBaram,

Could you do an english review please?

Thanks,
David.

File: app/_posts/2016-08-18-getting-started-with-a-core-initiative.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: "Getting started with a Core Initiative"
+tags:
+  - Drupal
+  - Contribution
+  - JSON API
+permalink: "/content/getting-started-with-a-core-initiative"
+layout: post
+image: "/assets/images/posts/getting-started-with-a-core-initiative/json-api.png"
+author: DavidHernandez
```
Comment (amitaibu) [inline app/_posts/2016-08-18-getting-started-with-a-core-initiative.md]: Can you add your data also [here](https://github.com/Gizra/gizra.com/blob/383a4253939cc05b83fff1a53900f9d0019bc69c/_config.yml#L37)

**Comment by amitaibu** [comment]
Looks good. Maybe @IshaDakota can have an English review as-well if Rachel is out.

**Comment by RachelBaram** [comment]
I can look over it on Sunday.

**Comment by IshaDakota** [comment]
@DavidHernandez did a [pull request](https://github.com/DavidHernandez/gizra.com/pull/1) on your fork.

**Comment by DavidHernandez** [comment]
Merged PR from @IshaDakota.
Added links to D.O user profiles.
Added changes to _config.yml requested by @amitaibu. This last change my add a dependency on this issue: https://github.com/Gizra/gizra.com/issues/430

**Comment by amitaibu** [comment]
Closed in favor of #481

**Pull request closed without merging**